F/FastANI/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+using BinaryBuilder, Pkg
+
+name = "FastANI"
+version = v"1.33.0"
+sources = [
+    GitSource("https://github.com/ParBLiSS/FastANI.git", "6531da0957344d2a56e3eeb9bb83fea02cc4abe0"),
+]
+
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/FastANI
+
+update_configure_scripts --reconf
+autoupdate
+
+./bootstrap.sh
+./configure \
+    --prefix=$prefix \
+    --build=${MACHTYPE} \
+    --host=${target} \
+    --with-gsl=${destdir} \
+    --includedir=${includedir} \
+    --libdir=${libdir}
+    
+if [[ $target == *-apple-* ]]; then
+    MACFLAGS="-stdlib=libc++"
```

Comment:
Why this?  Clang knows which is its C++ standard library.

Reply:
Mostly I'm following instructions for the how the developer said to install, I'll add a commit removing it

Reply:
Well, there is lots of nonsense there.